cli/wsproxy.ml

```diff
@@ -103,10 +103,16 @@ let proxy (fd : Lwt_unix.file_descr) protocol localport =
 let handler sock msg =
   Lwt_log.debug_f "Got msg: %s" msg >>= fun () ->
   match Re.Str.(split @@ regexp "[:]") msg with
+  | [protocol;_;path]
+  | [protocol;path] when String.contains path '/' ->
+    let addr = Unix.(ADDR_UNIX path) in
```

Comment:
We may want to restrict the paths that `wsproxy` can connect to for security, i.e. avoid connecting to message-switch, xapi socket, etc. directly, but that can be added later once we got the basics working.
I am thinking we could have a whitelist of directories, and you can only connect to  sockets under that directory (and not accepting .. in the path to avoid directory traversals).

Reply:
Maybe you could make it accept the list from an external configuration to keep it useful for other people